### PR TITLE
Add model gateway example (AI use case)

### PR DESCRIPTION
**Pull request by JayaniH**
## Purpose
Add example for model gateway https://github.com/wso2-enterprise/choreo/issues/25622

### CODE REVIEW

File: gateway-service/service.bal [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    serviceUrl = azureOpenAIServiceUrl
+);
+
+const string PII_DETECTION_ERROR_MESSAGE = "PII detected in the request payload";
+
+# A service representing a network-accessible API
+# bound to port `9090`.
+service / on new http:Listener(9090) {
+
+    resource function post [string deploymentId]/completions(@http:Payload text:Deploymentid_completions_body request) returns http:Response|text:Inline_response_200|error {
```
Comment (shafreenAnfar) [inline gateway-service/service.bal:49]: Can't we avoid using http:Response here?

Comment (JayaniH) [reply gateway-service/service.bal]: addressed in 7cc052e

File: gateway-service/service.bal [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    serviceUrl = azureOpenAIServiceUrl
+);
+
+const string PII_DETECTION_ERROR_MESSAGE = "PII detected in the request payload";
+
+# A service representing a network-accessible API
+# bound to port `9090`.
+service / on new http:Listener(9090) {
+
+    resource function post [string deploymentId]/completions(@http:Payload text:Deploymentid_completions_body request) returns text:Inline_response_200|http:BadRequest|error {
```
Comment (shafreenAnfar) [inline gateway-service/service.bal:49]: `text:Inline_response_200` shouldn't it be `text:InlineResponse200`?

Comment (shafreenAnfar) [reply gateway-service/service.bal:49]: Usual practise is to use PascalCase.

Comment (JayaniH) [reply gateway-service/service.bal:49]: @shafreenAnfar this is a type defined in the connector which was generated by the ballerina openapi tool

**Review by shafreenAnfar — APPROVED** [review]

**Comment by JayaniH** [comment]
This sample is added to the [choreo-samples](https://github.com/wso2/choreo-samples) repo instead (https://github.com/wso2/choreo-samples/pull/86)

**Pull request closed without merging**